### PR TITLE
Add type stubs for jcconv3 package

### PR DESCRIPTION
**Pull request by richardjharris**
Points of confusion:
 - input type (str | bytes) matches output type, but unsure how to
   express this
 - mode parameter of convert must be a valid jcconv.CONSTANT but
   for now I'm using integer literals

### CODE REVIEW

**Review by AlexWaygood — COMMENTED** [review]
We still support Python 3.6, so you'll have to use `typing_extensions.Literal` instead of `typing.Literal`

File: stubs/jcconv3/jcconv3/jcconv3.pyi [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from typing import Literal
```
Comment (AlexWaygood) [inline stubs/jcconv3/jcconv3/jcconv3.pyi:1]:
```suggestion
```

Comment (richardjharris) [reply stubs/jcconv3/jcconv3/jcconv3.pyi:1]: d'oh, thanks

**Comment by AlexWaygood** [comment]
It looks like this library probably meets our criteria for having stubs added to typeshed. At the same time, [the library doesn't appear to have been updated in over two years](https://github.com/LIQRGV/jcconv), and I'm not hugely enthusiastic about adding stubs for a library that's unmaintained.

**Comment by richardjharris** [comment]
Thanks for the input. I just found out I can put my own typings in a `typings` directory for personal use. That's enough for my needs so happy to close this.

**Comment by AlexWaygood** [comment]
> Thanks for the input. I just found out I can put my own typings in a `typings` directory for personal use. That's enough for my needs so happy to close this.

Thanks for understanding! Typeshed is a pretty huge repository, and adding any new stubs always adds a maintenance burden for us, so we do have to have some limits on what we accept :)

**Pull request closed without merging**